docs/API.md

```diff
@@ -2,6 +2,20 @@
 
 Percona Everest provides a set of APIs that enable you to access its features programmatically. These APIs provide a convenient and efficient way for developers to interact with the various functionalities of Percona Everest. 
 
+## Using the API
+
+The API is accessible on the same host and port as the Percona Everest Web UI. You can find detailed information about connecting to the Percona Everest Web UI in the [installation guides](install/installEverest). For instance, if you are connecting to your Percona Everest instance using port forwarding:
+
+```sh
+kubectl port-forward svc/everest 8080:8080 -n everest-system
+```
+
+You can then connect to the API via the `http://127.0.0.1:8080 endpoint`.
+
+### Authentication
+
+Currently, the API only supports authentication via a JWT token. You can obtain this JWT token from the Web UI after logging in. The JWT token is stored in the everestToken variable within the browser's local storage.
```

Comment:
> Also, is it true that I can connect to API with JWT only?

Yes, it's true. 

Regarding obtaining the token. Everest allows two type of users - the built-in users and the external IdP users. The described way of obtaining the token is correct for both types. However for the built-in users we can also use the `/session` endpoint to get the API token, example: 
```
curl --location -s 'localhost:8080/v1/session' --header 'Content-Type: application/json' --data '{"username": "your_usernmae","password": "your_password"}' | jq -r .token
```
@recharte do you have any concerns if we mention this approach in docs?

Reply:
I agree with Oksana
I think it would be more correct to mention the `/session` API as suggested because we don't really show the token anywhere on the UI, which means that the following sentence isn't very accurate:
> You can obtain this JWT token from the Web UI after logging in

However, I think it's fine to keep the mention that the token is stored in the local storage:
> The JWT token is stored in the everestToken variable within the browser's local storage.